tests/common/BundlerTest.cs

```diff
@@ -106,6 +106,63 @@ static void Main ()
 			}
 		}
 
+#if __MACOS__
+		// XM doesn't support removing the dynamic registrar yet.
+		//[TestCase (Profile.macOSMobile)]
```

Comment:
Is this still true? There is text above about XM not being able to guess but you can pass the command line arg to force it.

Reply:
Yes, this is still true, I found problems with it for XM quite late and didn't have to fix them before d15-7.